### ISSUE TITLE
openwifi use FDD mode

### ISSUE THREAD

**Issue by wangdan25** [open-sdr/openwifi#369]
openwifi currently uses FDD mode to realize simultaneous receiving and transmitting, but when AP is working, I find that their receiving and transmitting frequency Settings are the same. Isn't FDD requiring different receiving and transmitting frequencies? Won't interference be caused if the frequencies are the same?

**Comment by JiaoXianjun** [comment]
https://www.sciencedirect.com/science/article/abs/pii/S0140366423002244?via%3Dihub
https://arxiv.org/abs/2204.07354

**Comment by wangdan25** [comment]
Thank you for your answer. I have read this paper.But at present, openwifi uses the normal FDD mode, and the frequency of receiving and sending is the same, and it seems to work normally, here I do not understand how it is to achieve frequency division duplex

**Comment by wangdan25** [comment]
Thank you for your answer. I have read this paper.But at present, openwifi uses the normal FDD mode, and the frequency of receiving and sending is the same, and it seems to work normally, here I do not understand how it is to achieve frequency division duplex

**Comment by wangdan25** [comment]
Maybe I misunderstood, because I saw that the device tree parameter "adi,frequency-division-duplex-mode-enable" in the zedboard used by openwifi was enabled

**Comment by JiaoXianjun** [comment]
yes. you are right. but that does not matter. we set tx rx frequency to the same in the fdd mode.

**Comment by wangdan25** [comment]
I seem to understand some, combined with your paper, although the AD9361 register configuration uses FDD mode, but in the TX state will actually turn off the RX LO, so strictly speaking, it is not received and sent at the same time.
